common/fe-ste-models.go

```diff
@@ -595,6 +595,8 @@ func (TransferStatus) SkippedFileAlreadyExists() TransferStatus { return Transfe
 
 func (TransferStatus) SkippedBlobHasSnapshots() TransferStatus { return TransferStatus(-4) }
 
+func (TransferStatus) TierAvailabilityCheckFailure() TransferStatus { return TransferStatus(-5) }
```

Comment:
I just realised, adding or removing a transfer status requires changes to the aggregation logic in `GetJobSummary`.  But do we really want to do that?  What about just using the existing failure status?  Especially given that these types of failure will be rare.

Reply:
This actually is just one line.
![image](https://user-images.githubusercontent.com/49764384/73316365-830adf00-41e7-11ea-8dc5-364a34778dce.png)